src/controllers/user.controller.js

```diff
@@ -86,4 +87,35 @@ export class UserController {
       });
     }
   }
-}
+    static async getProfile(req, res){
+        try {
+            const user = await UserService.getUserById(req.user.id);
+            return res.status(200).json({
+                message: 'profile gotten successful',
+                data:user
+            });
+        }catch (e){
+            return res.status(500).json({error:'something went wrong', data:e,});
+        }
```

Comment:
Replicate the catch block from the register, and do it on other catch block you wrote